proof_ipa.go

```diff
@@ -449,6 +449,10 @@ func PreStateTreeFromProof(proof *Proof, rootC *Point) (VerkleNode, error) { //
 		}
 	}
 
+	if len(poas) != 0 {
```

Comment:
Considering in L419, `poas` is re-sliced on every usage this means that the final length must be zero.